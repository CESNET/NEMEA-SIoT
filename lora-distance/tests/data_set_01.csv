uint64 TIMESTAMP,double RSSI,string PHY_PAYLOAD
1504805915,-119,400910012600840501E410666602
1504719167,-117,40D12B0126007B4501A5469B48533D6D97C2230252C0A6BBE873202033D703533961
1504954329,-120,402929012680100201D71D4BDBA20705119A304E1710782914
1504836202,-115,40D12B0126007C4501BD4CA9CE935FB2E6E1CE8D0C3BDD20AD4F0451096A25652FF7
1504845353,-120,40EC13012680F57701895399EEB6A0
1504713807,-119,4009100126008605014EDA100BA7
1521568001,-115,7C561EAF51621B4B710A2D42E4963CE3080A151FD2F567666FB7BEA5E41B255245841EE25CC05503A1C75BE038DC200E869F0B0E0C9CBF01586A0E
1504883666,-116,40D12B0126007D4501D0391E4AC46092075E0F789EF3D19B130B23F3A591B9F38FB9
1504883666,-118,4009100126008705012018333C96
1504946087,-119,40D12B0126007E450138D08C539AD21CB50BDC8CCD0452EEF8E113BD01B95ABB6DC9
1504859758,-119,4009100126008805013F02960340
1504734234,-119,40D12B012600804501BAE2318377CC5CC9FC2B97EFD8AD7E9BA52D41DF43BF200747
1504850719,-118,4009100126008A05010A3DBF79AB
1504864424,-118,40EC13012680F67701523967E84489
1504943125,-117,40C910F1D056DB050170D788D162
1504715969,-111,4009100126008C050168845FDEA7
1504722946,-117,4009100126008D0501A294E6AA16
1504746875,-113,40D12B01260083450181570DDD4BF729D18F780BBB3224F5FD1FAFF1AFADC6F0F548
1504769231,-125,C52D3D40952D953C83DDB00E3F940C2FDCAF2BAC26F06CF944A382CB3062A06B8996DE7DDF774E26DF5C9C2DE6B83985C2BC752D
1504805909,-118,4009100126008E0501B2B06A5444
1504917703,-120,40D12B012600844501949CC6332E8D20D507EE772BA6E02B26CB9E0CEBD81ADD2356
1504703509,-116,4009100126008F0501676F40EC1C
1504830970,-120,EAAFA3841B1FEFF89D940061E36EC51A9F0617C5C278E4683CA8203A487B0022316ADD67C5833EA477F4CEBAF07C0CEDE68AD51DCB3116
1504855579,-113,40EC13012680F77701F532CC2B2518
1504738808,-118,2D3FD04EA2A880EDD1D303138FBACDBAE091C2054BC064E8E9711061E6E9A8
1504804556,-119,27895EC560F4BC5C3775A1BE34CA974C37F97F972A23FFAFF1F5680E354C4AF49D0C811BCFC917C8008EADBD191C5D51048B6DF10DEA77DC66CAD7236318AD7D696889B560A428DE04C1FDC78117BB75014B2774508E5F601440937E52813F6163CE0A2377BEA4ACCA7FC5DB649DCDC7EAF2F5883BE6704153033867406A4E125948F27A2C3F154495A9D214D20B75198C6B21B4DC69E8D24145CF0D7A1636EF2D4C89A355C912C57D9E4138BFB188FF431F71FD090AAC6E30EDF74BE9CDF88B892DCEF4CC20674B1DC3EE8595C52BC3117944306783F2955F9C1841F753B3A514050B5CCDC1AE4AC380971CEE5F7DF92D7805
1504857509,-120,400910012600900501616205DC57
1504802064,-119,40D12B012600864501AAD4704453E9FA36917A9A63263842C9A96CB535D1763FEB64
1504850594,-114,4009100126009105019E8B627680
1504817639,-118,40D12B01260087450184931B9F16E9F6B291CD2ACEEC36D17A718E5A1E5D793C2878
1504810225,-118,4009100126009205013FABF23C94
1504910275,-113,40D12B012600884501A11ACDC407B65A33C972B8084E48304C62071507BF887139CF
1504808730,-117,400910012600930501CA348CA234
1504856325,-117,516E88E34F9EF0BD1A210A75645FFD419ADFDF1315A01B2E03FE57EE88E39F9F4E761AA0EFC51553FC94FEAD58AE5027555391815001DAE5F7CEBEA681EBE0C570418AD00D3996483D918574BB2CF547AD033F41D34AE10D1C64ADC66E99624A92D6DA
1504797454,-119,A3FB974D9786EF01C96BE828F63A3CC3AA41482E82D3A22E693F3E7D4ABF8945600E28B84BB7C370864AB445584D39597A011423D671710FC94B470985E288CB2B9A0C436531FCE517E47E95750EB99475B0D8C868B00B30934FE13A6BF0F54EB37C8B7C2F9C7E2EB77AB358DE6A8E246FE3250AEF501A9687A2EB6591C4FB26852E6F1478054E2675D1B0D25798560E9FABF12AE5A97A6A8B9DA58889A542D2FF7A8100C42E108E68F6EC32A92903E1BCC6A8AA7B8CEE448FC197ADF267C7ED3C6AF686EF5E42331BDC85865BCA95DF68992D0F88D1179A
1504937388,-117,40D12B012600894501C0D081BF157A0B5D60C7D9E0C4A38D320EF9B5A4AD6BC67FF1
1504703260,-115,40091001260094050194AC01470A
1504703323,-119,40EC13012680F877017AC746873BE7
1504703385,-117,40091001260095050166C050A44D
1504703447,-116,40D12B0126008B4501CEF88317B68EA395F52D8A50ACA3C003C88270B314127B4E10
1504703634,-120,40091001260696050D09BF7AC93B
1504703696,-118,40D12B0126008C45016D6D3F861B87798E0569D8218B5F9399A196315169AAE28650
1504703759,-117,400910012600980501AF8632360C
1504703821,-120,40091001260099050141924773CF
1504703883,-119,4029290126801202018C0A38BBCAD2030E3CF7A57EB2E1E085
1504703946,-118,40D12B0126008E45012ADCA0249A27D865088950B08B3271752A230DEBE70B95948D
1504704008,-118,B438377FA4F9301E98C767CE2C097D9426D10DE0F58423C208698F397035D6A7283F8DF759E6D9C9519EAF645645E8F479F04B95B302A7AD6EA91976DDC00A8B170FEB9C65EDF35B4E41246822D637AE57FC245FE94E6B2044D10BC59CDE5B11CA778FB3F56A48006661718DA2BEA8EB2D78055F483409D30573AD332B000A42371411B53EB0EFABAA3E121596F4B2B07F5D61BA4DCB2147FB324CDD9FDDB1DEFA905BACDB
1504704070,-120,408A130126A0F977327F1C8B10F00C
1504704132,-120,4009100126009A0501026E521A5F
1504704195,-120,4009100126009B05012EE877DCE2
1504704257,-115,40D12B012600904501EA08D2CD3FD4ADC0CCFB91973FBC752418EC149AEA05668464
1504704506,-121,BAAB2C3A8454F4052BF52888A5AA52F646D0BE5F713BAB42CBC8E3E9D4F34F6B4E95E8C8B3C866324B3AEE9E728FC48D1FC42E5D8D0AA5A93068985E51FA4BFC014BFE9F7E90C3AA07DFE0C18F4C6CD958
1504704569,-115,40D12B0126009145018D4E40B1B3107D159EFFF8733CEF214F24E95A26C274CC6209
1504704693,-118,4009100126009D0501F83EE48C3E
1504704942,-120,4009100126009E0501BCA02A1301
1504705005,-117,40EC13012680FA7701B366C659BD56
1504705067,-118,4009100126009F0501B17AFECE01
1504705191,-115,40D12B012600944501CE43A9C7F59551096F18A6DB2331418DC244F4B4DDC22BEE7B
1504705254,-115,400910012600A00501F9195FFC2D
1504705316,-114,3EECD95C97BDA3596B84858E963EA862D2B27CDC6B004F8E8D32FA475107CB0A05F6EB638BDEB08CA0F197BF8B390FCAC59BE817B01914A363A6E4C4BBA39E16331334AE92FAA0B2D238B0C5A65DFFA2EAF91F4D68656018B00D20AB588795DBCC580798B17FD44D71C0ADBC345B10B1E0D8EA55699E4563E4D263228970CC03C53F23F79BDC99DAD231DA2A604A16C219194A0A690C912B311A803DF5A409A7305C54B6028816EC1C0E7C84D2C532FB54E5BB0B6C444D7AD1F1D84DC12DCC494E
1504705378,-119,40D12B01260096450137757BF31904AB9AB03704DC144FF248B445221E316FC55E61
1504705441,-118,40D12B0126009745010E3B46C799D6B8EFEB6A908F5BCD14BF903CC16BC95B405FE8
1504705503,-120,40EC13012680FB770125F0A61F1D72
1504705565,-119,40292901268013020108E1FB4F01CCDCC42C6959104FA23DFB
1504705628,-113,400910012600A405018EAC99B335
1504705690,-114,40D12B0126009845011C228E0F8F230CFFE45B3EB2F38B3B6E12A99435B24D168FD8
1504705752,-119,050D94C95A14F569A15683D2C6BE
1504705814,-119,AF1EA7F9F3C694A92DFBA953E8C5CD401EF05B9FC08A713ABDD169046DC6226CD4142892A480ABFD86C2851D18005E26A6457D95197DE3061FF8058A474186D6B78EB5E75F80A74A8AC35D36AB8AC81F94D994FD011B911211C3D02F04969D4980B65226A5BC4A2938087A6830937EB95A6CEAD40972BA197D8BAE04C659ED3A60160B31E4A1F2543B16A43D548A5AED151C60AEC9D839488973467A5ADFAABAF767650B32381577C13D30BD783AD473264EBFE1B9DEC68C217EE31FD1A296D55555497F3EF5BCD03BC73BE0D753E89262543B21CB059C
1504705877,-115,400910012600A605014859BDA665
1504706001,-117,1D819C6EDDD65B164D058AFC8D29DFF6A1861F1A083103356B21AA5E5B24F619A3FD949D482380665BAF2C9D906C87F2D27853
1504706064,-119,400910012600A705015DDE5275CD
1504706126,-118,40D12B0126009B8501B5FC0E73FB1D67B1CF042013280EB45D3B183AEAC1BE05711A
1504706188,-119,40EC13012680FC7701156C45177E42
1504706313,-116,400910012600A8050114B8920E2C
1504706375,-120,40C1B14E0180060008D776C57C0456596E828FE46059
1504706437,-113,400910012600A90501A460A66438
1504706500,-114,40D12B0126009D45013CC3DE7D8EBB65192464B385A3CB7225EB5D08CFCF73833725
1504706562,-117,400910012600AA05013316290201
1504706624,-120,16C0790115009B0501A9AAB2BBF0
1504706687,-119,40D12B0126009F450144306C7357FAB80632A910E5F8A21BE5442E238C6ADBF4612B
1504706749,-115,440D543516009C05015A0BDAFAE3
1504706811,-119,400910012600AD0501D04CE038A1
1504706874,-120,4029290126801402014C4767565BB1A9AA4B65D23A6241D6E2
1504706936,-119,AFD72081A36BBDADE593CC92CEBB7573E612FAE4084FC27AB2FE8E5D02BC995C709C5B4DCCB529FA9D3BE4D724D15A1A815D76789350AC5DA4292E6E5E
1504707060,-120,48B8530F16E4811625830965FF02C8011FB07689BE2D58A446F98DEEA5E46F558EBEB85D9E05245901DFA118B1C964A3CE207F33FDF08D31DDC330C4CC9218083C9C6DD75E7A065EF8033E0A42BA3C721C94D929AC
1504707123,-118,400910012600AE0501456C14F9DA
1504707185,-115,400910012600AF05018FD6905772
1504707247,-118,40D12B012600A24501A762DD179C04699026858780F1BE0C1FE67E93033568C5546E
1504707310,-118,400910012600B0050171A59E3044
1504707372,-120,48D12B412200E34501783A6CD3183BB42C6DB969584F701123F8D7330CB7BC2A6E05
1504707434,-119,40D12B012600A445017B96F00CB2A140418F7EC87DF51F0C7A7DB3320897256C7DBF
1504707497,-117,40EC13012680FE770177544C9D6325
1504707621,-120,400910012600B205016882EB708F
1504707683,-118,40D12B012600A54501907991DD01173D23DDBD288D243050410C3247B14CC802C6C8
1504707746,-119,400910012600B30501DEA2F04FC2
1504707808,-120,40D12B012600A645018C1475A5CF37D4C0D8323A50C63D5B796F5D67BBBDC6A70D2E
1504707870,-118,400910012600B405015E7371FC26
1504707933,-118,40D12B012600A74501BF5B94D049312BF16D86CEB88B54A04B4C2143A46A71662EF6
1504708057,-118,400910012600B505017EFB0AE669
1504708182,-121,425F9B14FAC6CEC59F3D4D451FC010A1C0204C08FE5898E0A66C16644025CA7A328C06522A0EF8DD28ADE57E6DFCD5506438A9468CBC42AC8FD9F6AF668B6DF1BF891B417AFBBE5CC23AC1F189440C2FDA1B7B55A5AFF683A8FF7EF17E7F9B36B2772278C8C8DF29E5
1504708369,-115,40D12B012600A84501CDB829BB2E1E8B4E237F9D461B891F7DACA11BF18EF92EBCBD
1504708431,-117,400910012600B605016C5FAA0D92
1504708493,-109,40EC13012680FF7701C0E25F236D3D
1504708556,-118,400910012600B70501BCCEED3A4B
1504708618,-120,402929012680150201A21D73E2294387996216EA6DC10C4023
1504708680,-118,40D12B012600AA450143B5D56674F6BC97D992312C7973595D884D544E33452BC651
1504708742,-118,400910012600B80501BC269661CF
1504708805,-119,45F918378719D448C975E2EEA9B1F6B96F50D62E82C2A687DE493F3159989760AC18A14599FFD162E879B75913D5A1F449421DAE876DCFC348A1B7588222A5843A6E8C24A304A89C0C4C803BEEE520
1504708867,-117,400910012600B90501E741D2BD8C
1504708929,-118,40D12B012600AC4501D226BA77C2B00978C4B26835A41C39A90CD6ADD60C50D0D648
1504708992,-119,400910012600BB0501FFEB302929
1504709241,-121,40D12B012600AD4501C05BA60E7822A6B63129F2779C7B6F001AF0DE85B0E8030064
1504709303,-119,40EC130126800078014FCF49394425
1504709365,-120,400910012600BC0501251735CC2A
1504709428,-114,40D12B012600AE45014636D04AC8385E2356C9F2EBCE37B4BD8B7D9849EA888D52D2
1504709552,-119,400910012600BD050161F8D275C1
1504709615,-120,867107DA7FAF0083433FDAFC3D7BE1095F6B7303F859ECD2D653871D88721BCB001C333DD90398A08947DC9814B6B2379B7455BAC01F0DB108FAD1C35C724BC3B1AC9C1762819ED301CC7340C4C7BF57DE95A881F5B0F5A363743E77C61014DCEFF97D91DF45AD591DC32BC8968E16AAB8394808D18DF975C980F4C4FED074EB9CC0CE2557982807F36D90798965A0676087675B9662780E1C4D9A014F0EF0E8FFA676EF1E1643F6E6CC07934F9FF6ABABA1739F896739D50EF240BC328377755CFB7AB2B52C
1504709677,-118,40D02B000600AF4501C2B093D07BEF0C5853FCEB6A6DC4DE997D45E365470E293258
