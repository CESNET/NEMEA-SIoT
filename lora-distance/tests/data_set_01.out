-125.000000,0.000000,0,0.100000,"06002BD0"
